mapsims/cmb.py

```diff
@@ -0,0 +1,81 @@
+import healpy as hp, numpy as np
+from pixell import enmap, curvedsky
+
+def get_cmb_sky(iteration_num, 
+                nside = None, #set this if healpix is desired
+                shape = None, #set shape and wcs if CAR maps are desired
+                wcs = None,
+                lensed = True,
+                aberrated = False,
+                pol = True,
+                cmb_set = 0, #We allow for more than one CMB map per lensing map
+                cmb_dir = None, 
+                nfreqs = 1):
+    '''
+    Return a CMB map from stored alm's.  This can be in Healpix format
+    (if nside is specified) or rectangular pixel format (if wcs and shape are
+    specified).  The lensed alm's are pre-stored.
+    If rectangular, it returns a stack of enmaps of shape (nfreqs, ncomp, ny, nx). 
+    If Healpix, it will return a numpy array of shape (nfreqs, ncomp, npix) 
+
+    Args:
+        iteration_num: integer specifying which sim iteration to use
```

Comment:
I recommend to use `numpy` style docstrings, see https://numpydoc.readthedocs.io/en/latest/format.html, like `healpy`

Reply:
The SO developer guide currently recommends "Google" style. Does numpy style render nicely in readthedocs? Happy to switch for this project of course.

Reply:
Not really, other than Google style being better for short docstrings (and faster to type out, since I don't have to type out underlines). Both seem to render nicely in readthedocs with the napolean extension. I'll switch to numpy style for this repo; and we should discuss changing the recommendation in the SO dev guide since the argument that pydata adopts it is compelling. Pinging @mhasself .

Reply:
Pushed numpy style.